### DISCUSSION TITLE
why can't change field attributes with clickhouse database

### DISCUSSION THREAD

**Discussion by flyly0755** [dbeaver/dbeaver#14180] [DBeaver Development]
In compare,  can change field attributes with postgresql database.
What is the different process logic with these 2 databases?

**Comment by LonwoLonwo** [comment]
Hello @flyly0755 
Please tell more about field attributes with examples.

**Reply by flyly0755** [reply]
I mean the field attributes of clickhouse table can't change with dbeaver,
for example the field _field1_ data type is uint32, now i want to change it to uint64, but can't . double click the  datatype cell then don't emerge the drop-down key for changing.
but for postgresql database, there isn't this problem can change it successfully.

**Comment by LonwoLonwo** [comment]
It's because we have not done this yet :)
You can create ticket as feature request here: https://github.com/dbeaver/dbeaver/issues

**Reply by flyly0755** [reply]
thank you, got it:)
